llvm/tools/llvm-cas-mc-dump/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(LLVM_LINK_COMPONENTS
+  Support
+  CAS
+  CASObjectFormats
+  )
+
+add_llvm_tool(llvm-cas-mc-dump
+  llvm-cas-mc-dump.cpp
```

Comment:
Is there a good reason not to call this a more generic `llvm-cas-dump`?

Reply:
It rely very heavily on this concept of "Schemas", and I am not sure if the base class interface of a Schema is generic enough for this tool. I'll check and report back

Reply:
I second Adrian's comment, wouldn't `llvm-cas-dump` be better since we just want to dump out the cas contents?